applications/tari_validator_node/proto/dan/consensus.proto

```diff
@@ -58,9 +59,10 @@ message HotStuffTreeNode {
 
 message ValidatorMetadata {
   bytes public_key = 1;
-  bytes signature = 2;
-  bytes merkle_proof = 3;
-  uint64 merkle_leaf_index = 4;
+  bytes shard_id = 2;
```

Comment:
did you mean vn_shard_key ?

Reply:
Ah ok wasn't clear on the naming of shard_id vs shard_key will update